src/backend/joanie/core/api/admin.py

```diff
@@ -71,6 +186,34 @@ class CertificateDefinitionViewSet(viewsets.ModelViewSet):
     filter_backends = [django_filters.rest_framework.DjangoFilterBackend]
     filterset_class = filters.CertificateDefinitionAdminFilterSet
 
+    def delete(self, request, *args, **kwargs):
```

Comment:
Maybe we could write a mixin to factorize this method then replace `viewsets.ModelViewSet` by mixins.